### PR TITLE
[update] ssr adapter astro add instructions

### PR DESCRIPTION
**Pull request by sarah11918**
- New or updated content

Docs currently only show manual instructions for adding SSR adapters. This updates all the (located in Docs repo) pages to first instruct using `npx astro add` and then offer the current manual instructions (install the package, update `astro.config.mjs`) as an alternative.

Pages updated: SSR and all the deploy pages. (Integration package READMEs and `configuration-reference` will need to be updated at `withastro/astro`.)

NOTE: Current Maintainers discussion needs to be resolved first re: our general move to recommend `dependencies` over `devDependencies` because of `astro add`'s behavior. These changes assume that manual installation instructions should NOT show installing as dev dependencies, to be consistent. But, since `astro add` currently DOES use the dev flag, we want to wait for a resolution before publishing these instructions. 

POSSIBLE CHANGES NEEDED: manual instructions might need to have the `--save-dev` flag added back if we decide these ARE to be installed as devDeps

### CODE REVIEW

File: src/pages/en/guides/deploy/netlify.md [hunk 1]
```diff
@@ -20,31 +20,37 @@ Your Astro project is a static site by default. You don’t need any extra confi
...
+      npm install @astrojs/netlify
     ```
 
 1. Add two new lines to your `astro.config.mjs` project configuration file.
 
-    ```diff
+    ```js title="astro.config.mjs" ins={2, 5-6}
     import { defineConfig } from 'astro/config';
-    + import netlify from '@astrojs/netlify/functions';
+    import netlify from '@astrojs/netlify/serverless';
```
Comment (sarah11918) [inline src/pages/en/guides/deploy/netlify.md:41]:
```suggestion
    import netlify from '@astrojs/netlify/functions';
```

File: src/pages/en/guides/server-side-rendering.md [hunk 2]
```diff
@@ -24,11 +24,19 @@ The following adapters are available today with more to come in the future:
...
+
+You can add any of the official adapters with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step. For example, to install the Netlify adapter, run:
+
+```bash
+npx astro add netlify
+```
+
+You can also add an adapter manually by installing the package and updating `astro.config.mjs` yourself. (See the links above for adapter-specific instructions to complete the following two steps to enable SSR.) Using `my-adapter` as an example placeholder, the instructions will look something like:
+
 1. Install the adapter to your project dependencies via npm or your package manager of choice
```
Comment (sarah11918) [inline src/pages/en/guides/server-side-rendering.md:36]:
```suggestion
1. Install the adapter to your project dependencies  using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```

File: src/pages/en/guides/deploy/vercel.md [hunk 3]
```diff
@@ -31,22 +31,29 @@ There is currently a Vercel issue displaying a 404 page on Astro websites. Until
...
-1. Install [the Vercel adapter](https://github.com/withastro/astro/tree/main/packages/integrations/vercel) to your project’s dependencies.
+Add [the Vercel adapter](/en/guides/integrations-guide/vercel/) to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
+
+```bash
+npx astro add vercel
+```
+
+If you prefer to install the adapter manually instead, complete the following two steps:
+
+1. Install [the `@astrojs/vercel` adapter](/en/guides/integrations-guide/vercel/) to your project’s dependencies.
```
Comment (sarah11918) [inline src/pages/en/guides/deploy/vercel.md:42]:
```suggestion
1. Install [the `@astrojs/vercel` adapter](/en/guides/integrations-guide/vercel/) to your project’s dependencies using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```

File: src/pages/en/guides/deploy/netlify.md [hunk 4]
```diff
@@ -20,31 +20,37 @@ Your Astro project is a static site by default. You don’t need any extra confi
...
-1. Install [the Netlify adapter](https://github.com/withastro/astro/tree/main/packages/integrations/netlify) to your project’s dependencies.
+Add [the Netlify adapter](/en/guides/integrations-guide/netlify/) to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
+
+```bash
+npx astro add netlify
+```
+
+If you prefer to install the adapter manually instead, complete the following two steps:
+
+1. Install [the `@astrojs/netlify` adapter](https://github.com/withastro/astro/tree/main/packages/integrations/netlify) to your project’s dependencies.
```
Comment (sarah11918) [inline src/pages/en/guides/deploy/netlify.md:31]:
```suggestion
1. Install [the `@astrojs/netlify` adapter](https://github.com/withastro/astro/tree/main/packages/integrations/netlify) to your project’s dependencies using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```

File: src/pages/en/guides/deploy/deno.md [hunk 5]
```diff
@@ -21,7 +21,15 @@ Your Astro project can be deployed to [Deno Deploy](https://deno.com/deploy) as
...
-1. Install [the Deno adapter](https://github.com/withastro/astro/tree/main/packages/integrations/deno) to your project’s dependencies.
+Add [the Deno adapter](/en/guides/integrations-guide/deno/) to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
+
+```bash
+npx astro add deno
+```
+
+If you prefer to install the adapter manually instead, complete the following two steps:
+
+1. Install [the `@astrojs/deno` adapter](https://github.com/withastro/astro/tree/main/packages/integrations/deno) to your project’s dependencies.
```
Comment (sarah11918) [inline src/pages/en/guides/deploy/deno.md:32]:
```suggestion
1. Install [the `@astrojs/deno` adapter](https://github.com/withastro/astro/tree/main/packages/integrations/deno) to your project’s dependencies using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```

File: src/pages/en/guides/deploy/cloudflare.md [hunk 6]
```diff
@@ -64,14 +64,20 @@ It's then possible to update the preview script in your `package.json` to `"prev
...
+Add the Cloudflare adapter to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
 
-1. Add the [`@astrojs/cloudflare` adapter](https://github.com/withastro/astro/tree/main/packages/integrations/cloudflare#readme) to your project's `package.json` by running:
+```bash
+npx astro add cloudflare
+```
+
+If you prefer to install the adapter manually instead, complete the following two steps:
+
+1. Add the `@astrojs/cloudflare` adapter to your project's `package.json` by running:
```
Comment (sarah11918) [inline src/pages/en/guides/deploy/cloudflare.md:77]:
```suggestion
1. Add the `@astrojs/cloudflare` adapter to your project's dependencies using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```

File: src/pages/en/guides/server-side-rendering.md [hunk 7]
```diff
@@ -24,11 +24,19 @@ The following adapters are available today with more to come in the future:
...
+
+You can add any of the official adapters with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step. For example, to install the Netlify adapter, run:
+
+```bash
+npx astro add netlify
+```
+
+You can also add an adapter manually by installing the package and updating `astro.config.mjs` yourself. (See the links above for adapter-specific instructions to complete the following two steps to enable SSR.) Using `my-adapter` as an example placeholder, the instructions will look something like:
+
+1. Install the adapter to your project dependencies  using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```
Comment (Yan-Thomas) [inline src/pages/en/guides/server-side-rendering.md:36]: Extra whitespace
```suggestion
1. Install the adapter to your project dependencies using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for this @sarah11918, I added a small suggestion! I'll hold my "LGTM" till we have an decision about `dependencies` or `devDependencies`.

File: src/pages/en/guides/server-side-rendering.md (hunk 7 above)
Comment (sarah11918) [reply src/pages/en/guides/server-side-rendering.md]: Thanks, for the awesome catch, Yan! That is exactly where I was pasting the entire 2nd part of that phrase. You caught me! I *thought* I got a single space every time...

**Pull request merged**